x-pack/plugins/fleet/server/services/epm/elasticsearch/transform/install.ts

```diff
@@ -115,17 +116,27 @@ async function handleTransformInstall({
   callCluster: CallESAsCurrentUser;
   transform: TransformInstallation;
 }): Promise<EsAssetReference> {
-  // defer validation on put if the source index is not available
-  await callCluster('transport.request', {
-    method: 'PUT',
-    path: `/_transform/${transform.installationName}`,
-    query: 'defer_validation=true',
-    body: transform.content,
-  });
-
+  try {
+    // defer validation on put if the source index is not available
+    await callCluster('transport.request', {
+      method: 'PUT',
+      path: `/_transform/${transform.installationName}`,
+      query: 'defer_validation=true',
+      body: transform.content,
+    });
+  } catch (err) {
+    // swallow the error if the transform already exists.
+    const isAlreadyExistError =
+      isLegacyESClientError(err) && err?.body?.error?.type === 'resource_already_exists_exception';
+    if (!isAlreadyExistError) {
+      throw err;
+    }
+  }
   await callCluster('transport.request', {
     method: 'POST',
     path: `/_transform/${transform.installationName}/_start`,
+    // Ignore error if the transform is already started
+    ignore: [409],
```

Comment:
Same as above, cc @kevinlog

Reply:
same here, looks OK - will check with @pzl

Reply:
as long as we don't think we're swallowing other problems that might be in a 409, sure